app/facet_selector/facet_selector.controller.js

```diff
@@ -12,6 +12,8 @@
       $scope.transfers = Transfer;
       $scope.tags = Tag;
 
+      $scope.date_regex = '\\d\\d\\d\\d([-\/]\\d\\d)?([-\/]\\d\\d)?';
```

Comment:
Two-digit years are too ambiguous, in my opinion - let's disallow them.

If you can believe it, `Date.parse` doesn't tread `'2005-01-01'` and `'2005-1-1'` as the same thing... but it seems to be misinterpreting the zero-prefixed version, so yeah, let's allow the one-digit dates. 😩